string node_name
float64 publish_rate_in_hertz
uint64 sequence_number
float64 local_utc_timestamp